mappings/net/minecraft/menu/Menu.mapping

```diff
@@ -1,4 +1,19 @@
-CLASS net/minecraft/class_1703 net/minecraft/container/Container
+CLASS net/minecraft/class_1703 net/minecraft/menu/Menu
+	COMMENT A menu is a list of items and properties, synchronized between the server and the client.
```

Comment:
To address @sfPlayer1's concern at https://github.com/FabricMC/yarn/pull/846#issuecomment-540034825, we should point out that this is not a menu in computing sense at all.